client/src/animation/fadeout.js

```diff
@@ -0,0 +1,12 @@
+import { keyframes } from '@emotion/core';
+
+const fadeout = keyframes`
```

Comment:
```suggestion
const fadeOut = keyframes`
```
and the filename `fadeOut.js`